### PR TITLE
Feature per file build time

### PR DESCRIPTION
**Pull request by a-vlad**
<b>Description:</b> Add checkbox that shows per file total build time. This is an aggregate of all function build times.

screen shot 2016-08-17 at 1 37 20 pm

### CODE REVIEW

File: BuildTimeAnalyzer/CMResultWindowController.swift [hunk 1]
```diff
@@ -22,9 +22,14 @@ class CMResultWindowController: NSWindowController {
     @IBOutlet weak var buildDurationTextField: NSTextField!
     @IBOutlet weak var cancelButton: NSButton!
     @IBOutlet weak var searchField: NSSearchField!
+    @IBOutlet weak var perFileCheckbox: NSButton!
```
Comment (gprasant) [inline BuildTimeAnalyzer/CMResultWindowController.swift]: why is this an NSButton instead of a checkbox ?

Comment (a-vlad) [inline BuildTimeAnalyzer/CMResultWindowController.swift]: With OSX checkboxes / radio ext are all NSButton with different type. 

screen shot 2016-08-17 at 2 01 24 pm

**Comment by RobertGummesson** [comment]
Hi @a-vlad 

Sorry, it's been taking so long to respond. Been busy. I just had a look at this and it looks great. Good idea with the per file times. 

One comment though, would it be possible to alert the user when the CSV file has been created? I pressed it but without the feedback, I wasn't quite sure what happened. Had to check the code to learn that it was saved to my desktop.

I can have a look at this myself when I have time if you don't.

**Comment by a-vlad** [comment]
This PR only has per file aggregation. Can open new PR from my Master branch which has the save feature and I can add a dialogue to indicate where file has been saved.

**Comment by RobertGummesson** [comment]
Ah right. That's good then. Merged.

**Pull request merged**